src/dialogs/more-info/const.ts

```diff
@@ -111,7 +112,8 @@ export const computeShowLogBookComponent = (
 
```

Comment:
There is a check for `stateObj.attributes.unit_of_measurement` a couple of lines up that would become redundant and should be removed as `isNumericFromAttributes` already checks that